scripts/corr2_dsim_control.py

```diff
@@ -12,32 +12,57 @@
 
 from corr2.dsimhost_fpga import FpgaDsimHost
 
+
 parser = argparse.ArgumentParser(description='Control the dsim-engine (fake digitiser.)',
                                  formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
 parser.add_argument('--config', type=str, action='store', help=
+
                     'corr2 config file. (default: Use CORR2INI env var)')
+
 parser.add_argument('--program', dest='program', action='store_true', default=False,
                     help='(re)program the fake digitiser')
+
 parser.add_argument('--deprogram', dest='deprogram', action='store_true', default=False,
                     help='deprogram the fake digitiser')
+
 parser.add_argument('--start', dest='start', action='store_true', default=False,
                     help='start the fake digitiser transmission')
+
 parser.add_argument('--pulse', action='store_true', default=False, help=
                     'Send a pulse of fake packets. Does nothing if digitiser is '
                     'already transmitting, use --stop first. It will only work '
                     'once until the dsim-engine is reset, e.g using --resync '
                     'which can be used with --pulse (--resync will be applied first).')
+
 parser.add_argument('--pulse-packets', action='store', default=100, help=
                     'Send out out this many data packets per polarisation if --pulse '
                     'is specified (default: %(default)d.')
+
 parser.add_argument('--resync', action='store_true', default=False,
                     help='Restart digitiser time and then sync')
+
 parser.add_argument('--stop', dest='stop', action='store_true', default=False,
                     help='stop the fake digitiser transmission')
+                    
 parser.add_argument('--loglevel', dest='log_level', action='store', default='',
                     help='log level to use, default None, options INFO, DEBUG, ERROR')
+
 parser.add_argument('--ipython', action='store_true', default=False, help=
                     'Launch an embedded ipython shell once all is said and done')
+
+parser.add_argument('--set_sine_source', nargs = 3, default=False, help=
```

Comment:
Also, I would prefer it to be --set-sine-source since the rest of the corr2 utils always use -, not _, and the same for the other options you added.